rust/constants/Cargo.toml

```diff
@@ -1,12 +1,19 @@
 [package]
 name = "constants"
 version = "0.1.0"
-authors = ["vera"]
-edition = "2018"
+edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+hex = {version = "0.4.3", optional = true}
```

Comment:
looks like `hex` can be a [dev dependency](https://doc.rust-lang.org/rust-by-example/testing/dev_dependencies.html) so there would be no need for this optional feature

Reply:
`test_values.rs` contents are used in other crates tests (`db_handling`, `navigator`, `transaction_parsing`, `transaction_signing`), so the `hex` propagation is needed, I suppose. Maybe we should rename the feature from `test` to `integration_test` or something, it's really confusing.